style.css

```diff
@@ -547,3 +547,494 @@ hr {
+  }
+
+  .hambuger {
+    display: none;
+  }
+
+  .logo {
+    width: 92px;
+    height: 20px;
+    left: 24px;
+    color: white;
+    font-size: 20px;
+    font-family: "inter", sans-serif;
+    margin-left: 154px;
+    text-decoration: none;
+  }
+
+  .menu {
+    display: flex;
+    padding: 20px 154px 0 0;
+    font-family: 'inter', sans-serif;
+  }
+
+  .menu .mail-link {
+    margin-top: 3px;
+  }
+
+  a {
+    color: white;
+    text-decoration: none;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  .menu-link {
+    padding-right: 30px;
+  }
+
+  .menu-link2 {
+    padding-right: 30px;
+  }
+
+  .menu-link3 {
+    padding-right: 30px;
+  }
+
+  .hambuger .logo {
+    display: none;
+  }
+
+  .primary-text {
+    margin-bottom: 12px;
+  }
+
+  .status-bar {
+    display: none;
+  }
+
+  .topnav {
+    display: flex;
+    justify-content: space-between;
+    flex-direction: row;
+    align-items: center;
+    width: 1349px;
+    background-color: rgb(60, 58, 57);
+    height: 72px;
+  }
+
+  h1 {
+    color: hsl(25, 100%, 50%);
+    font-family: "Crete Round", sans-serif;
+    font-weight: normal;
+    font-size: 72px;
+    line-height: 81px;
+  }
+
+  .supporting-text {
+    font-family: "inter", sans-serif;
+    font-weight: bold;
+    font-size: 24px;
+    line-height: 32px;
+    color: hsl(40, 100%, 94%);
+    margin-bottom: 12px;
+  }
+
+  .description {
+    display: block;
+    width: 566px;
+    font-family: "inter", sans-serif;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 20px;
+    line-height: 32px;
+    color: rgba(244, 245, 247, 1);
+  }
+
+  .headline {
+    display: flex;
+    background: rgba(28, 26, 1) url(./images/Illustration-header.png) no-repeat;
+    background-position: right;
+    height: 798px;
+    padding: 0;
+    width: 1349px;
+    margin: auto;
+  }
+
+  .hero {
+    padding: 143px;
+  }
+
+  .social-icons {
+    position: absolute;
+    display: flex;
+    flex-direction: column;
+    left: 40px;
+    bottom: 187px;
+  }
+
+  .social-icons li {
+    margin: 0;
+  }
+
+  /* work-section */
+
+  .work-section {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 120px 0 0 143px;
+  }
+
+  .work-header {
+    width: 315px;
+    font-size: 40px;
+    font-weight: 400;
+    font-family: "Crete Round", serif;
+    line-height: 52px;
+    padding: 20px 0;
+  }
+
+  .btn-card {
+    cursor: pointer;
+    display: inline-block;
+    border: 1px solid;
+    padding: 6px 12px;
+    background: white;
+    color: black;
+    font-size: 15px;
+    line-height: 24px;
+    text-decoration: none;
+    margin-right: 10px;
+  }
+
+  .btn-car {
+    cursor: pointer;
+    display: inline-block;
+    border: 0;
+    padding: 10px 10px;
+    background: rgb(255, 255, 255, 0.24);
+    color: white;
+    font-size: 15px;
+    line-height: 24px;
+    text-decoration: none;
+    margin-right: 10px;
+  }
+
+  .btn-card:hover {
+    opacity: 0.9;
+  }
```

Comment:
- [ ] You've implemented some `:hover` effect, Although the hover you've implemented doesn't match the Figma design. Since we are trying to match the Figma as closely as possible please make sure to implement the correct button interaction states as per the Figma template
- [ ] Make sure to also implement the pressed state (that is while the button is being clicked) using CSS `:active` pseudo-class.
- [ ] kindly implement these states on all the buttons on the page

## Figma button interactions
![image](https://user-images.githubusercontent.com/18730106/145456647-9caf855d-bd78-4d24-9ac3-c62cffe55806.png)